src/store/ChartState.js

```diff
@@ -501,6 +501,15 @@ class ChartState {
         if (grab && this.stxx.chart.lockScroll) {
             this.stxx.chart.lockScroll = false;
         }
+        if (grab && this.stxx && this.stxx.chart) {
+            const dataSegment = this.stxx.chart.dataSegment;
+            if (dataSegment) {
+                const startPointOnChartData = dataSegment[0];
+                const barsDisplayedOnScreen = Math.floor(this.stxx.chart.width / this.stxx.layout.candleWidth);
```

Comment:
@nazaninreihani You can use `this.stxx.chart.maxTicks`. Its value is as same as `barsDisplayedOnScreen` and it shows the number of bars(ticks) on the screen.